charts/prometheus-elasticsearch-exporter/ci/security-context.yaml

```diff
@@ -1,5 +1,5 @@
 ---
-# Set default security context for kubernetes
+# Set default security context for Openshift
```

Comment:
Why change to support OpenShift only?

Reply:
Hopefully my adaption resolves the confusion and at the same time better mirrors the effect the previous version of this file had.